### PR TITLE
Cosmos Resources

### PR DESCRIPTION
**Pull request by Michelle-L**

### CODE REVIEW

File: resources/cosmos.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Cosmos Hub
+
+The Cosmos Hub (`gaia`) docs have moved [here](https://github.com/cosmos/gaia/tree/master/docs).
+
+## Languages
+
+The Cosmos-SDK is currently written in [Golang](https://golang.org/), though the framework could be implemented similarly in other languages. Contact us for information about funding an implementation in another language.
+
+
```
Comment (ligi) [inline resources/cosmos.md]: why so many new lines at the end of the file?

**Review by Michelle-L — COMMENTED** [review]
I removed them I hope. First time :)

**Comment by Michelle-L** [comment]
cc @mariapaulafn

**Comment by soc1c** [comment]
please also link it in the resouces.md otherwise people wont find it

**Pull request merged**